test/IDE/structure_object_literals.swift

```diff
@@ -13,3 +13,10 @@ struct I: _ExpressibleByImageLiteral {
 
 // CHECK: <gvar>let <name>z</name>: I? = <object-literal-expression>#<name>imageLiteral</name>(<arg><name>resourceName</name>: "hello.png"</arg>)</object-literal-expression></gvar>
 let z: I? = #imageLiteral(resourceName: "hello.png")
+
+_ = .<#placeholder#>
```

Comment:
These shouldn't appear in the document structure, correct? We'd just be testing whether it affects the literal's appearance?